client/layout/masterbar/style.scss

```diff
@@ -3,9 +3,14 @@ $autobar-height: 20px;
 
 // The WordPress.com Masterbar
 .masterbar {
-	background: $masterbar-color;
-	border-bottom: 1px solid darken( $masterbar-color, 4% );
-	color: $white;
+	border-bottom: 1px solid;
+	@include theme-multiple(
+		(
+			'background': --masterbar-bg-color,
+			'border-color': --masterbar-border-color,
+			'color': --masterbar-color,
+		)
+	);
```

Comment:
This seems adding opacity to the simple assignment of a variable. Do we really need a special syntax instead of simply three lines? What's the benefit?

```
background: var( --masterbar-bg-color  );
border-color: var( --masterbar-border-color  );
color:  var( --masterbar-color  );
```